README.md

```diff
@@ -69,36 +69,49 @@ Setting up Fabrik on your local machine is really easy. You can setup Fabrik usi
 
 
 ### Using Virtual Environment
-1. First set up a virtualenv
+1. First set up a virtualenv. Fabrik runs on Python2.7.
     ```
     sudo apt-get install python-pip python-dev python-virtualenv
-    virtualenv --system-site-packages ~/Fabrik
+    virtualenv --system-site-packages ~/Fabrik --python=python2.7
     source ~/Fabrik/bin/activate
     ```
 
-2. Clone the repository
+2. Clone the repository via git
     ```
-    git clone --recursive https://github.com/Cloud-CV/Fabrik.git
+    git clone --recursive https://github.com/Cloud-CV/Fabrik.git && cd Fabrik
     ```
 
 3. Rename settings/dev.sample.py as settings/dev.py and change credential in settings/dev.py
     ```
     cp settings/dev.sample.py settings/dev.py
     ```
-    Replace the hostname to ``` localhost ``` in settings/dev.py line 14.
+    * Replace the hostname to ``` localhost ``` in settings/dev.py line 14.
+    It should now look like this:  
```

Comment:
`line 14` change this to `line 15`

Reply:
Done :+1:

Reply:
Also add a new line. It doesn't look good in markdown